packages/react/src/components/Authenticator/shared/__tests__/FormFields.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from '@testing-library/react';
+import * as React from 'react';
+
+import { defaultFormFieldOptions } from '@aws-amplify/ui';
+import { FormField, FormFieldProps } from '../FormField';
```

Comment:
Prefer to name test files based on the file they are testing. As we are only testing the `FormField` component and not the `FormFields` think we can name this file _FormField.text.tsx_, what do you think?

Reply:
Oops, that's a typo ;)

Reply:
84fd456